src/views/Project/index.jsx

```diff
@@ -356,23 +376,28 @@ export default class Project extends Component {
       [...issues, ...goodFirstBugs, ...mentoredBugs],
       'summary'
     );
+    const quickIntroduction = project.introduction.split(/(?=##\s)/)[0];
+    const fullIntroduction = project.introduction;
+    const source = this.state.open ? fullIntroduction : quickIntroduction;
+    const buttonText = this.state.open ? 'See Less' : 'See More';
```

Comment:
No need to store `buttonText`, `source`, `fullIntroduction` and `quickIntroduction` in variables since they're not being re-used.